.rubocop.yml

```diff
@@ -11,6 +11,13 @@ AllCops:
     - db/schema.rb
     - vendor/bundle/**/*
     - node_modules/**/*
+    - tmp/**/*
```

Comment:
I don't really like listing all this crud here, but it does noticeably speed up rubocop.